roles/variables/defaults/main.yml

```diff
@@ -203,7 +203,8 @@ confluent_cli_version: 3.63.0
 ### Recommended replication factor, defaults to 3. When splitting your cluster across 2 DCs with 4 or more Brokers, this should be increased to 4 to balance topic replicas.
 default_internal_replication_factor: 3
 
-### SASL Mechanism to set on all Kafka Listeners. Configures all components to use that mechanism for authentication. Possible options none, kerberos, plain, scram, scram256
+### SASL Mechanism to set on all Kafka Listeners. Configures all components to use that mechanism for authentication. Possible options none, kerberos, plain, scram, scram256.
+### You can provide a comma-separated list of mechanisms to configure multiple listeners with different mechanisms. For example, 'kerberos,plain'
```

Comment:
should we also add info as to how the protocol between bk-bk or kraft-kraft or bk-kraft be picked. And some details like how the first protocol in this list will be chosen at places?

Reply:
Updated the description. Not adding too much information since it will confuse the users.